docs/PUMLFiles/Review/review_class.puml

```diff
@@ -0,0 +1,15 @@
+@startuml
+
+'review1, class diagram'
+
```

Comment:
If I'm not wrong, the notation for this module requires the Class circle icon to be hidden. 

It can be hidden on PlantUML with `hide circle`  at the top of the file.